WordPress/Classes/ViewRelated/Stats/Charts/SparklineView.swift

```diff
@@ -86,26 +99,27 @@ class SparklineView: UIView {
         gradientLayer.frame = bounds
 
         // Calculate points to fit along X axis, using existing interpolated Y values
-        let segmentWidth = bounds.width / CGFloat(data.count-1)
-        let points = data.enumerated().map({ CGPoint(x: CGFloat($0.offset) * segmentWidth, y: $0.element) })
+        let segmentWidth = bounds.width / CGFloat(chartData.count-1)
```

Comment:
perhaps you should do a check on chartData since it could result in divide by 0?  (note you flagged this on one of my PR's rightly so!) 🙂

Reply:
Thanks for catching this!